packages/client-recommendation/src/types/GetPersonalizationStrategyResponse.ts

```diff
@@ -2,22 +2,22 @@ export type GetPersonalizationStrategyResponse = {
   /**
    * Events scoring
    */
-  readonly eventsScoring: ReadonlyArray<{
-    readonly eventName: string;
-    readonly eventType: string;
-    readonly score: number;
+  eventsScoring: ReadonlyArray<{
```

Comment:
this is still ReadOnlyArray, not Array

Reply:
Good catch – I checked with a regex now and it seems to be fine!